java/core/src/java/org/apache/orc/impl/WriterImpl.java

```diff
@@ -273,6 +277,15 @@ private static int getClosestBufferSize(int size) {
     return Math.min(kb256, Math.max(kb4, pow2));
   }
 
+  static {
+    try {
+      com.github.luben.zstd.util.Native.load();
+    } catch (UnsatisfiedLinkError | ExceptionInInitializerError e) {
+      LOG.warn("Unable to load zstd-jni library for your platform... " +
```

Comment:
Minor nit

Can we use single statement and avoid  "..."

Unless it violates our checkstyle rules, something like
```
LOG.warn("Unable to load zstd-jni library for your platform. Using builtin-java classes where applicable");
```